integration_tests/src/main/python/ast_test.py

```diff
@@ -87,6 +87,34 @@ def test_null_literal(spark_tmp_path, data_gen):
     assert_gpu_ast(is_supported=True,
                    func=lambda spark: spark.read.parquet(data_path).select(f.lit(None).cast(data_type)))
 
+@pytest.mark.parametrize('data_descr', [
+    (boolean_gen, True),
```

Comment:
NIT: Can we create a variable for this list to avoid duplicate code in these two tests ?